src/components/Tooltip/tooltip.js

```diff
@@ -0,0 +1,27 @@
+import Tooltip from '@material-ui/core/Tooltip';
+import { withStyles } from '@material-ui/core';
+
+const tooltip = withStyles((theme) => ({
+  tooltip: {
+    backgroundColor: '#FFFFFF',
```

Comment:
Can you make a basic style schema read from props so that we can use it for different commons?